### DISCUSSION TITLE
Nginx Configuration Parse Error

### DISCUSSION THREAD

**Discussion by GyimPet** [0xJacky/nginx-ui#162] [General]
Error message: Nginx Configuration Parse Error unexpected token Eof () on line 13, column 1.

But there is nothing in line 13....:

![2023-09-05 10_06_19-Edit Site _ Nginx UI – Mozilla Firefox](https://github.com/0xJacky/nginx-ui/assets/77054641/4e94c037-9281-4889-8141-a96e7d44a008)

**Comment by 0xJacky** [comment]
Can you provide us the full content of your configuration file? You can hide the sensitive information, and we this need to reproduce the issue.

**Reply by GyimPet** [reply]
Okay. How exactly should I send it? email ?

**Reply by 0xJacky** [reply]
Just comment on this discussion

**Reply by GyimPet** [reply]
Ok.

My system: Raspberry PI 3 B+, OP: DietPi RPi-ARMv8-Bookworm, Nginx UI version: v1.9.9-4 (and for test: v2.0.0-beta.3) 
I use two websites; 'default' (no problem) and 'remote' (faulty).

I renamed both files to .TXT and uploaded them

[default.txt](https://github.com/0xJacky/nginx-ui/files/12525560/default.txt)
[remote.txt](https://github.com/0xJacky/nginx-ui/files/12525561/remote.txt)

Otherwise, the Nginx-UI works fine, but I can't edit the 'remote' file here, only directly in the folder with the nano editor. I can easily change the 'default' pages, no problem there. The 'nginx -t' command does not return any errors and nginx works fine.

**Reply by 0xJacky** [reply]
ok, I will check it tomorrow.

**Reply by 0xJacky** [reply]
It seems like a issue of gonginx, if you remove this part of content, then the parser can work.
```
limit_req_zone $binary_remote_addr zone=req_per_ip:1m rate=30r/m;
limit_conn_zone $binary_remote_addr zone=conn_per_ip:1m;
client_body_buffer_size 200K;
client_header_buffer_size 2k;
client_max_body_size 200k;
large_client_header_buffers 3 1k;
```

**Reply by GyimPet** [reply]
Indeed, if I delete this part, it works without a problem! 
What's wrong with these commands? Are they inappropriate? 
I just started working with Nginx, any advice is welcome!

Thanks for the quick help !
